test/f90_correct/src/acl04.f90

```diff
@@ -0,0 +1,27 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! Test for array constructor whose element is array of dynamic char.
+
+program m_test_adjust_len
+  call test_adjust_len(4, 4, 5)
+  print *, "PASS"
+
+contains
+  subroutine test_adjust_len(a, b, c)
+    integer :: a, b, c
+    character(len=a) :: s1(2)
+    character(len=b) :: s2
+    character(len=c) :: s3(1)
+
+    s2 = "aaaa"
+    s3 = "bbbb"
```

Comment:
To make the intent of the test more explicit, I suggest passing 6 to the parameter `c`, and setting `s3` to `"bbbbbb"`.

Reply:
Thanks. Fixed.